internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -579,6 +585,26 @@ func resourceRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Di
 	return nil
 }
 
+// getReplicationSpecIDsFromOldAPI returns the id values of replication specs coming from old API. This is used to populate old replication_specs.*.id attribute avoiding breaking changes.
+// In the old API each replications spec has a 1:1 relation with each zone, so ids are returned in a map from zoneName to id.
+func getReplicationSpecIDsFromOldAPI(ctx context.Context, projectID, clusterName string, connV220231115 *admin20231115.APIClient) (map[string]string, error) {
+	clusterOldAPI, _, err := connV220231115.ClustersApi.GetCluster(ctx, projectID, clusterName).Execute()
+	if apiError, ok := admin20231115.AsError(err); ok {
+		readErrorMsg := "error reading  advanced cluster with 2023-02-01 API (%s): %s"
+		if apiError.GetErrorCode() == "ASYMMETRIC_SHARD_UNSUPPORTED" {
+			tflog.Info(ctx, fmt.Sprintf(readErrorMsg, clusterName, err))
```

Comment:
are these logs shown in Prod?   not sure about logging non-error or non-warning info

Reply:
yes they would be shown, path that is being logged is not too relevant so simply removing